DOCUMENTATION.md

```diff
@@ -271,6 +275,66 @@ with open('file.csv', 'rb') as f:
     }
 ```
 
+#### Ask recipients to confirm their email address before they can download the file
+
+This feature is currently opt-in only. From 29 March 2023 it will apply to all new files by default, unless you opt out.
+
+##### Opt in
+
+To ask recipients to confirm their email address, set the `verify_email_before_download` flag to `true`.
+
+You will not need to do this after 29 March 2023.
+
+```python
+from notifications_python_client import prepare_upload
+
+with open('file.pdf', 'rb') as f:
+    ...
+    personalisation={
+      'first_name': 'Amala',
+      'application_date': '2018-01-01',
+      'link_to_file': prepare_upload(f, verify_email_before_download=True),
+    }
+```
+
+##### Opt out (not recommended)
+
+You should not opt out if you send files that contain personally identifiable information or sensitive information.
```

Comment:
```suggestion
You can turn this feature off on a per-file basis. You should not do this if you send files that contain either:

* personally identifiable information
* commercially sensitive information
```

Reply:
Same comment about updating the API client as above

Reply:
I'm interested in why we decided to include `information classified as ‘OFFICIAL’ or ‘OFFICIAL-SENSITIVE’ under the Government Security Classifications policy` please? Was this a suggestion from IA or privacy?

And should we link to the Government Security Classifications policy here?